docs/source/pre_application/cloud_node_management.rst

```diff
@@ -11,26 +11,12 @@ PRE Node Cloud Automation
     via `nucypher-ops <https://github.com/nucypher/nucypher-ops>`_.
 
 
-.. warning::
-
-    `nucypher-ops <https://github.com/nucypher/nucypher-ops>`_ is under active development, and
-    should currently **only be used on testnet**.
-
-
-PRE Node Testnet Cloud Automation
-=================================
-
-In this tutorial we're going to setup a Threshold PRE Node on the rinkeby testnet using a remote cloud provider (Digital Ocean, AWS, and more in the future).
+In this tutorial we're going to setup a Threshold PRE Node using a remote cloud provider (Digital Ocean, AWS, and more in the future).
 Whilst this example will demonstrate how to deploy to Digital Ocean, the steps for any other infrastructure provider are virtually identical.
 There are a few pre-requisites before we can get started.
 First, we need to create accounts at `Digital Ocean <https://cloud.digitalocean.com/>`_ and `Infura <https://infura.io>`_.
-We also need at least 40,000T in our wallet.
-There are currently no Threshold testnet faucets but if you ask in the `Discord <https://discord.gg/Threshold>`_ channel, someone will be happy to help you out.
-
-.. warning::
-
-  Ensure that you are using a wallet on the rinkeby testnet, **don't** use a mainnet address.
 
+In order to run a node will also need to have an existing Threshold stake.  For more information see
```

Comment:
You don't need an existing stake, you can do all the setup via nucypher-ops and only need to stake as the very last step.